tensorflow/examples/ios/caption/tensorflow_utils.h

```diff
@@ -0,0 +1,52 @@
+// Copyright 2015 Google Inc. All rights reserved.
```

Comment:
If possible, could you reference this file from it's original location rather than duplicating it? Same goes for the .mm below.

Reply:
In `tensorflow_utils.mm`, I changed the config because it was causing run-time crashes:
```objective-c++
options.config.set_inter_op_parallelism_threads(1);
options.config.set_intra_op_parallelism_threads(1);
```

Would you prefer if the session options were adjusted elsewhere?